src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.html

```diff
@@ -37,3 +37,12 @@
     </span>
   </span>
 </ng-template>
+
+<ng-template #editableTpl let-value="value">
```

Comment:
I would suggest to use the yesNoText column that will be introduced with #25989 (see https://github.com/ceph/ceph/pull/25989/files#diff-be3e8c59695fc2d22701cff36f26dd63R8 and https://github.com/ceph/ceph/pull/25989/files#diff-246efec6cbd1bfb84c053e1067c8b65bR186).

Reply:
I generally favor the approach where things are generalized, but this does not nearly look like the current implementation. I suggest to do that generalization and refactoring later and talk about how it should look afterwards, as I think people might want to have it look more like it currently does.